script/helperFunctions.sh

```diff
@@ -1507,16 +1507,32 @@ function verifyContract() {
   fi
 
   echo "[info] trying to verify $CONTRACT on $NETWORK with address $ADDRESS using Sourcify now"
-  forge verify-contract \
-    "$ADDRESS" \
-    "$CONTRACT" \
-    --chain-id "$CHAIN_ID" \
-    --verifier  sourcify
+  if isZkEvmNetwork "$NETWORK"; then
+    FOUNDRY_PROFILE=zksync ./foundry-zksync/forge verify-contract \
+      "$ADDRESS" \
+      "$CONTRACT" \
+      --zksync \
+      --chain-id "$CHAIN_ID" \
+      --verifier sourcify
```

Comment:
in foundry.toml you have "verifier: zksync" and here you use sourcify. Is that intended and correct?
Could add a comment to describe what is behind this.

Reply:
This is the fallback verification. This only attempts to run if the normal verification fails.